configs/usaid-laos-ev.nrel-op.json

```diff
@@ -77,6 +77,6 @@
         "map_trip_lines": false,
         "push_send": false,
         "options_uuids": true,
-        "options_emails": true
```

Comment:
@niccolopaganini since `usaid-laos-ev` should not be changed, it should not even appear in the list of modified files. Please revert correctly.

As an aside, I am not sure that this is valid JSON given the trailing `,`.